### PR TITLE
Align diagram placement description on the labels' one

### PR DESCRIPTION
**Pull request by DelazJ**
(these settings are the same so let's use same description) and remove some redundant/obsolete information

### CODE REVIEW

File: docs/user_manual/style_library/label_settings.rst [hunk 1]
```diff
@@ -519,9 +519,10 @@ Priority
 ........
 
 In the :guilabel:`Priority` section you can define the placement priority rank
-of each label, ie if there are different labels candidates for the same
-location, the label with the higher priority will be displayed and the
+of each label, ie if there are different diagrams or labels candidates for the
+same location, the label with the higher priority will be displayed and the
```
Comment (havatv) [inline docs/user_manual/style_library/label_settings.rst:523]:
```suggestion
same location, the item with the higher priority will be displayed and the
```
?

File: docs/user_manual/working_with_vector/vector_properties.rst [hunk 2]
```diff
@@ -1795,18 +1795,26 @@ placement (more details at :ref:`Placement <labels_placement>`):
...
-detect and solve position conflicts between diagrams and labels by setting
-the :guilabel:`Priority` slider value.
+  In that case, QGIS will look for the optimal position of the diagram.
+  Remember that you can also use the line orientation for the position
+  of the diagram.
+* :guilabel:`Around centroid` (at a set :guilabel:`Distance`),
+  :guilabel:`Over centroid`, :guilabel:`Using perimeter` and
+  :guilabel:`Inside polygon` are the options for polygon features.
+
+The :guilabel:`Coordinate` group provides direct control on diagrams
```
Comment (havatv) [inline docs/user_manual/working_with_vector/vector_properties.rst:1805]:
```suggestion
The :guilabel:`Coordinate` group provides direct control on diagram
```

File: docs/user_manual/working_with_vector/vector_properties.rst [hunk 3]
```diff
@@ -1795,18 +1795,26 @@ placement (more details at :ref:`Placement <labels_placement>`):
...
+placement, on a feature-by-feature basis, using their attributes
```
Comment (havatv) [inline docs/user_manual/working_with_vector/vector_properties.rst:1806]:
```suggestion
placement, on a feature-by-feature basis, using attributes
```

Comment (DelazJ) [reply docs/user_manual/working_with_vector/vector_properties.rst:1806]: I'd keep this one, to enforce the option to use their own attributes. When using others' attributes, you are in the expression case.

Comment (havatv) [reply docs/user_manual/working_with_vector/vector_properties.rst:1806]: "Their" points to the diagrams. Do the diagrams have attributes? If not, "their" must be replaced by something else ("using the feature's attributes"?).

**Pull request merged**